### PR TITLE
Added Java code snippets for Bloom #3630 

### PR DESCRIPTION
**Pull request by Ranjeet0611**
Added Java code snippets for Bloom #3630

### CODE REVIEW

**Comment by Ranjeet0611** [comment]
Hi @dwdougherty  Please check this build works for you
Thank you

**Review by dwdougherty — APPROVED** [review]
LGTM. Thank you, Ranjeet.

File: src/test/java/io/redis/examples/BloomFilterExample.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        System.out.println(res2); // >>> True
+
+        boolean res3 = unifiedJedis.bfExists("bikes:models", "Smoky Mountain Striker");
+        System.out.println(res3); // >>> True
+
+        List<Boolean> res4 = unifiedJedis.bfMAdd("bikes:models",
+                "Rocky Mountain Racer",
+                "Cloudy City Cruiser",
+                "Windy City Wippet");
+        System.out.println(res4); // >>> True
```
Comment (sazzad16) [inline src/test/java/io/redis/examples/BloomFilterExample.java:32]: res4 is a list of booleans. So the output can't only be `True`.

**Review by sazzad16 — CHANGES_REQUESTED** [review]
@dwdougherty Weren't the `import`s supposed to be removed or at least hidden?

@Ranjeet0611 Here is only one object of UnifiedJedis. So use a shorter variable name; like jedis, client, etc.

File: src/test/java/io/redis/examples/BloomFilterExample.java (hunk 1 above)
Comment (Ranjeet0611) [reply src/test/java/io/redis/examples/BloomFilterExample.java]: @sazzad16 fixed it 
Thank you

**Comment by Ranjeet0611** [comment]
@sazzad16  Please check now.
Thank you

File: src/test/java/io/redis/examples/BloomFilterExample.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// EXAMPLE: bf_tutorial
+
+// HIDE_START
+package io.redis.examples;
+
+import redis.clients.jedis.UnifiedJedis;
+import org.junit.Test;
+import org.locationtech.jts.util.Assert;
```
Comment (sazzad16) [inline src/test/java/io/redis/examples/BloomFilterExample.java:8]: Use `junit.Assert`.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/test/java/io/redis/examples/BloomFilterExample.java (hunk 2 above)
Comment (Ranjeet0611) [reply src/test/java/io/redis/examples/BloomFilterExample.java:8]: Sure

**Comment by Ranjeet0611** [comment]
@dwdougherty  please review

**Comment by dwdougherty** [comment]
@sazzad16 This PR builds correctly and it's good to go.

**Review by dwdougherty — APPROVED** [review]

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**